src/components/StateListItem.vue

```diff
@@ -153,12 +153,7 @@
 
 .state-list-item__name { @apply
   text-base
-  text-subdued
-  shrink-0
-  whitespace-nowrap
-  grow-0
-  text-ellipsis
-  overflow-hidden;
+  text-subdued;
```

Comment:
If we're allowing wrapping here we should align anything else on the line to the top of the bounding box; as it is right now, tags will align to the middle of the content instead, which breaks scanning expectations (see your screenshot for a minor example of this)

Reply:
Good call. Here's an updated screenshot:

<img width="879" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6776415/57daa417-5793-44a2-ada4-d5a3a0f2b3d5">

Reply:
Alignment when it's not wrapping:

<img width="886" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6776415/f53aeefc-3f70-4a9a-882d-90bfd3deeaa6">